src/components/tables/TableItems.tsx

```diff
@@ -311,12 +312,16 @@ export const ReviewStatus = ({ data }: { data: ReviewTranscript }) => {
 
 export const GroupedLinks = ({ data }: { data: ReviewTranscript }) => {
   const { pr_url } = data.review!;
-  let publishUrl = "";
   const isPublished = data.review?.mergedAt;
+  const newTranscriptsUrl =
+    data.transcriptUrl && resolveGHApiUrl(data.transcriptUrl);
+  let publishUrl = "";
 
   if (isPublished) {
-    let fileSlug = deriveFileSlug(data.content.title);
-    publishUrl = derivePublishUrl(fileSlug, data.content.loc);
+    let fileSlug = deriveFileSlug(data.content.title, /:/);
+    publishUrl = newTranscriptsUrl
+      ? derivePublishUrl(fileSlug, newTranscriptsUrl.srcDirPath)
+      : derivePublishUrl(fileSlug, data.content.loc);
```

Comment:
Thanks for this, I did a test on both and it gave the same output which is below
Older transcripts
<img width="831" alt="Screenshot 2024-04-22 at 13 55 01" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/90271995/11018c45-6ef2-4ec9-b8e6-16ebc6409f33">
Newer transcripts
<img width="831" alt="Screenshot 2024-04-22 at 13 56 53" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/90271995/a90f04a4-fad4-4a4f-b89c-8942667128bf">

I want to know , how you found out the current code doesn't work, or is it based on making the logic more simpler

Reply:
It wasn't working because you were using the transcript title for creating the slug. but that's not always true for the new implementation. 

That's not on you, it's more of a general issue of us not documenting correctly how things work. Another reminder of how functional tests could be helpful on catching this kind of issues.

Reply:
Oh right , Thanks.